template/apiExamples/basic.html

```diff
@@ -0,0 +1 @@
+<[namespace]-[name]>Hello World</[namespace]-[name]>
```

Comment:
I feel like this file should be in the `./demo` dir. Placing this dir at the root level of the repo gives the appearance that this has an overarching role in the repo, when its primary role is with the demo docs.

Reply:
I thought about that but this file can be used in multiple ways, including importing and using the API code examples directly in the automated tests.

Reply:
Where will individual doc partials go? I don't want to get into the pattern of creating a lot of root-level directories for the breaking up of md docs. Previously you had this in a `./docs/topics/` dir.

Reply:
I did have all the topics in that sub dir and I still plan to. But I also had the code examples separated from the docs just like I did here. There just aren't any partials in this stripped down release.

Reply:
This will be fine for now. But when we get into the overall content strategy of things, we may need to revisit this.